packages/nexus-sdk/src/__tests__/utils.spec.ts

```diff
@@ -39,4 +45,24 @@ describe('utils functions', () => {
       expect(removeLeadingSlash('*')).toEqual('*');
     });
   });
+
+  describe('buildHeader()', () => {
+    it('should return a header string', () => {
+      expect(buildHeader('json')).toEqual('application/ld+json');
+      expect(buildHeader('vnd.graph-viz')).toEqual('text/vnd.graphviz');
+      expect(buildHeader('n-triples')).toEqual('application/n-triples');
+    });
+  });
+
+  describe('parseAsBuilder()', () => {
+    it('should return a parseAs value', () => {
+      expect(parseAsBuilder('json')).toEqual('json');
+      expect(parseAsBuilder('vnd.graph-viz')).toEqual('text');
+      expect(parseAsBuilder('blob')).toEqual('blob');
+      expect(parseAsBuilder('document')).toEqual('document');
+      expect(parseAsBuilder('arraybuffer')).toEqual('text');
+      expect(parseAsBuilder('n-triples')).toEqual('text');
+      expect(parseAsBuilder('stream')).toEqual('text');
```

Comment:
Can you add a test with a random string as args, like `buildHeader('lol')` and make sure it defaults to json?